aequilibrae/utils/spatialite_utils.py

```diff
@@ -38,7 +38,9 @@ def connect_spatialite(path_to_file: os.PathLike, missing_ok: bool = False) -> C
 
         return qgis.utils.spatialite_connect(path_to_file)
 
-    ensure_spatialite_binaries()
+    if os.environ.get("AEQ_ENSURE_SPATIALITE", "YES") != "NO":
```

Comment:
I think this should be 
```suggestion
    if os.environ.get("AEQ_ENSURE_SPATIALITE", "YES") == "YES":
```
or
```suggestion
    if os.environ.get("AEQ_ENSURE_SPATIALITE", "NO") == "YES":
```

Otherwise things like `n`, `N`, `0`, and `no` will get interpreted as a "yes".